docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -1,49 +1,55 @@
+= Write a synthetic test
 
 ++++
-<titleabbrev>Create a test</titleabbrev>
+<titleabbrev>Write a synthetic test</titleabbrev>
 ++++
 
-[discrete]
 [[synthetics-syntax]]
-== Syntax
 
 beta[] To write synthetic tests for your application, you'll need to know basic JavaScript and
 {playwright-url}[Playwright] syntax.
-The Elastic Synthetics agent exposes an API for creating and running tests:
+The synthetics agent exposes an API for creating and running tests, including:
 
-* `journey(name, ({ page, browser, context, params }) => {})` -- A journey tests one discrete unit of functionality.
-For example, logging into a website, adding something to a cart, or joining a mailing list.
-Each journey provides a fresh playwright `browser`, `context`, and `page` instance.
-See <<synthetics-create-journey>> for more information.
-* `step(name, function)` -- A journey consists of multiple steps, i.e., actions that should be completed in a specific order.
-Steps are displayed individually in the {uptime-app} for easy debugging and error tracking.
-* `beforeAll(({ params }) => {})` -- Runs a provided function prior to all `journey` runs.
-If the provided function is a promise, the runner will wait for the promise to resolve before invoking the journey.
-This function is useful for setting up global state or a server that will be used across multiple journeys.
-* `before(({ params }) => {})` -- Runs a provided function prior to a single `journey` runs.
-This function is useful for setting up global state or a server that will be used for a single journey.
-* `afterAll(({ params }) => {})` -- Runs a provided function after all the journeys have completed.
-This function is useful for removing global state or closing a server that was used in multiple journeys.
-* `after(({ params }) => {})` -- Runs a provided function after a single journey has completed.
-This function is useful for removing global state or closing a server that was used in a single journey.
+[horizontal]
+`journey`::     Tests one discrete unit of functionality.
+                Takes two parameters: a `name` (string) and a `callback` (function).
+                See <<synthetics-create-journey>>.
+`step`::        Actions within a journey that should be completed in a specific order.
+                Takes two parameters: a `name` (string) and a `callback` (function).
+                See <<synthetics-create-step>>.
+`beforeAll`::   Runs a provided function before all `journey` runs.
+                If the provided function is a promise, the runner will wait for the
+                promise to resolve before invoking the `journey`.
+                Takes one parameter: a `callback` (function).
+                See <<before-after>>.
+`before`::      Runs a provided function before a single `journey` runs.
+                Takes one parameter: a `callback` (function).
+                See <<before-after>>.
+`afterAll`::    Runs a provided function after all the `journey` runs have completed.
```

Comment:
```suggestion
`afterAll`::    Runs a provided function once, after all the `journey` runs have completed.
```

I think it's important to include "once" for these hooks so it's clear they don't run for _each_ journey.